tests/configs/.rr-empty-global.yaml

```diff
@@ -8,6 +8,9 @@ server:
   relay: "pipes"
   relay_timeout: "20s"
 
+# Key must be defined if using AWS defaults, but can be empty.
+sqs: { }
+
```

Comment:
This is currently required because the "inside AWS" logic was removed and can then no longer be used to skip missing sqs config key. If this still needs to be supported, we have to change the code that detects missing configuration.

Reply:
We can't break configuration (or introduce some breaking changes) in minor releases. So the code needs to be updated to handle this case w/o BC.